userdocs/profiles.dev/docs/tutorial-basics/setup.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 1
+---
+
+# Environment setup
+
+:::info Assumed knowledge
+
+This tutorial assumes you have some knowledge of the concept of GitOps and are comfortable using
+[Flux](https://fluxcd.io/).
+
+Please refer to the [Introduction](/docs/intro) to read about the core concepts of Profiles.
+:::
+
+
+
+In this tutorial you will create and install a simple profile onto your Kubernetes cluster using various GitOps tools.
+
+_If you are not interested in writing profiles, just installing them, please skip ahead to the relevant section
+once you have set up your environment._
+
+------------------
+
+## Prerequisites
+
+In order to install profiles, you need to have the following set up:
+
+### Kubernetes cluster
+
+For local testing, we recommend using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+The cluster must be version 1.16 or newer.
+
+### Profiles CRDs and Flux CRDs
+
+While profiles can be installed manually, it is recommended to install them in a GitOps fashion.
+Profiles uses Flux to do this.
+
+To install both the profiles CRD and the required Flux CRDs, run:
+
+```bash
+pctl prepare
+```
+
+This will install the latest version of the profiles CRD, which may not always be stable.
+To specify a [specific version](https://github.com/weaveworks/profiles/releases), use the `--version` flag.
+
+### A GitHub repo, synced to Flux
+
+This tutorial will require a GitHub account. (More git providers will be added soon.)
+
+The repo can be public or private (note: you will not be asked to push any sensitive information) and must
```

Comment:
Also mention that your local setup needs to have access if it's private?

Reply:
that will be on install right? is there a flag or a token? i can;t see anything which looks right in `pctl install --help` 🤔

Reply:
@Skarlso ^^

Reply:
Crap, sorry, I missed this reply somehow.
It's also required for private profile repositories where the profile.yaml lives.
It's an implicit requirement, there are no flags for it. We assume that the user's environment is set up correctly for accessing all private repositories in a profile setting. Including nested profiles in possible different repositories.

Reply:
ah okay, so just saying something like "make sure your local git environment is set up correctly if you want to use private repos... etc" should be enough